dlc-wallet/src/main/scala/org/bitcoins/dlc/wallet/DLCWallet.scala

```diff
@@ -661,12 +691,33 @@ abstract class DLCWallet extends Wallet with AnyDLCHDWalletApi {
           updatedDLCDb <-
             updateFundingOutPoint(dlcDb.contractIdOpt.get, outPoint)
         } yield updatedDLCDb
-      case None =>
-        throw new RuntimeException(
-          s"No DLC Offer found with corresponding tempContractId ${accept.tempContractId.hex}, this wallet did not create the corresponding offer")
+      case (dlc, Some(_)) =>
+        logger.debug(
+          s"DLC Accept (${dlc.paramHash.hex}) has already been registered")
```

Comment:
to be consistent in logs, should probably either replace with contract id or at least add it to the message